CourseGrab/Supporting/Assets.xcassets/Logos/appdev-logo-white.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
```

Comment:
Could you make this single scale? This might change the sizing of the image in the launch screen so just be sure to re-size it.